README.md

```diff
@@ -1,94 +1,131 @@
 <p align="center">
-    <img src="logo.svg" alt="Node Cron Alarm Clock Star Logo" width="100" />
+  <img src="logo.svg" alt="cron for Node.js logo" height="150">
+  <br />
+  <b>cron</b> is a robust tool for running jobs (functions or commands) on schedules defined using the cron syntax.
+  <br />
+  Perfect for tasks like data backups, notifications, and many more!
 </p>
 
-# node-cron
+# Cron for Node.js
 
-[![Version](https://badgen.net/npm/v/cron?icon=npm)](https://badgen.net/npm/v/cron)
-[![Build Status](https://badgen.net/github/status/kelektiv/node-cron?icon=github)](https://badgen.net/github/status/kelektiv/node-cron)
-[![Build Checks](https://badgen.net/github/checks/kelektiv/node-cron?icon=github)](https://badgen.net/github/checks/kelektiv/node-cron)
```

Comment:
This was changed in @sheerlox 's commit so he can respond here.

Reply:
- the "Build Status" & "Build Checks" are merged in the single "Build Status" badge
- minified / minzipped sizes seemed irrelevant because the library is for Node.js, where most of the time package sizes aren't a concern, and when it is (for example in lambda cloud functions) the relevant metric is the unpacked size (which is available on the npm page)
 
regarding the Snyk badge, we (maintainers) have no access to its alerts, and the badge is static and doesn't provide more information than "monitored" / "unknown", as you can see below for a 10-year-old branch of `express`:
[![Snyk](https://snyk.io/test/github/expressjs/express/benchmark/badge.svg)](https://snyk.io/test/github/expressjs/express/benchmark/badge.svg)
![image](https://github.com/kelektiv/node-cron/assets/11234273/27045198-6ef2-447a-abb3-42b1029bbd1b)
and here's the head of the result of running [`osv-scanner`](https://osv.dev/) in that branch:
![image](https://github.com/kelektiv/node-cron/assets/11234273/9eceba12-be3b-4551-bc95-eff9cf861f73)

if we want to re-introduce that badge, here's an equivalent (static) badge in green and using the Shields project so it fits better with the other badges, wdyt?
[![Vulnerabilities](https://img.shields.io/badge/vulnerabilities-monitored-green)](https://img.shields.io/badge/vulnerabilities-monitored-green)

we could also replace it with a Renovate badge since it does the same job of finding vulnerabilities in dependencies:
[![Renovate](https://img.shields.io/badge/renovate-enabled-green)](https://img.shields.io/badge/renovate-enabled-green)